packages/starrating/src/react/index.js

```diff
@@ -85,6 +86,7 @@ class StarRating extends React.PureComponent {
       } else {
         if (this.isInteractive && !hasValidRating) {
           appearance = Star.appearances.empty
+          bright = true
```

Comment:
So, the star is "bright" when it's clickable and hasn't been rated yet...  Lots of rules here.

Reply:
srsly. that's why i started writing unit tests